### ISSUE TITLE
MAXIMO Mock Training 

### ISSUE THREAD

**Issue by rebekkabry** [cityofaustin/atd-data-tech#16061]
Review currently prepared MAXIMO training material, in a mock field crew training method, that would result in hands-on user feedback to support MAXIMO assessment and training initiatives. 
#14962 

**Recommended Steps** 
- [x] Meet with Adry to get vision for MAXIMO training initiatives.
- [ ] Review training materials and provide feedback on information. Meeting 3/6/24
- [x] Get feedback from Rene's experience. 
- [ ] Test retention and consider best practice Q/A methods. 
- [ ] Provide feedback to Adry and MAXIMO support team.
- [ ] Where should and which resources should accessed? Online, inside tool?   

**Questions** 
- [ ] Where will these documents live? 

**Retention Plan**
- [ ] MAXIMO Account Creation
- [ ] MAXIMO Basic Navigation 
- [ ] MAXIMO Workgroup Specific Needs 
- [ ] MAXIMO Workorder Handling 
- [ ] MAXIMO Report Pulls 
- [ ] EzMAXMobile Mirror

**Comment by rebekkabry** [comment]
### Recommendations and Insights 
_running list_

- [ ]  Update Pre-Training Correspondences with new links (ie. emails) {added 2/28/24}

**Comment by dianamartin** [comment]
# 3/7/2024
**Sample Training Invite**
![image](https://github.com/cityofaustin/atd-data-tech/assets/134445095/3fef93c7-c10d-4905-ba6a-561105160bec)

## Assumptions
- We thought this email was for a brand new Maximo user, but the email seems to indicate that this might be a follow-up training. The third training in fact. 
- After we finish the training, what does a user get after the training? (Ex. Resources, links, materials)

## Questions
- Was the first round of training cover the same material?
- What information is in the first couple of trainings?

**Comment by rebekkabry** [comment]
Creating New Work Orders Work session with Diana/Rebekka
- we need a new guide. This guide is for child work orders from parent. 

**Questions:** 
- Step 1: Create a Work Log: Where do we go to start step 1? We found it, but only because Diana had previous knowledge. 

We gonna pivot and try this one: SBO Maximo QRS 311 SR Screening WO Creation Field Op Staff
- Use this document
- We tried this [QRG first: Create New Parent Work Order](https://teams.microsoft.com/l/message/19:meeting_YzJlNTU2NTUtNWE1MC00Y2I1LTkxNjYtMDQ3ZTVkN2EwMzJi@thread.v2/1711046080323?context=%7B%22contextType%22%3A%22chat%22%7D)
- but there was a note:
  -  _Audience: Users already familiar with MAXIMO whose projects require that they create work orders with parent-child relationships. These 
instructions assume that (1) users are starting out with a child work order, and then (2) creating a parent work order from that record._


**Notes:** 
1. They are missing "how to create a work order" or is the work already done? What about coordinators that need to convert one? 
2. Many documents with "create new" which is most useful?
